### ISSUE TITLE
Invalid Object (Discount)

### ISSUE THREAD

**Issue by James-Azzopardi** [culturekings/afterpay#16]
Receiving error 

`[discounts[0].displayName may not be empty (was null), discounts[0].displayName may not be null (was null)]`

With trace: 

```
[discounts:protected] => Array
                                        (
                                            [0] => CultureKings\Afterpay\Model\Discount Object
                                                (
                                                    [displayName:protected] => Promocode Used: KAYLA.A.SHANAHAN0921080010
                                                    [amount:protected] => CultureKings\Afterpay\Model\Money Object
                                                        (
                                                            [amount:protected] => 37.99
                                                            [currency:protected] => AUD
                                                        )

                                                )

                                        )
```

**Comment by jared-fraser** [comment]
Mapping was missing for the discounts object.

**Comment by jared-fraser** [comment]
Fixed and tagged as v0.1.5
